metadata.rb

```diff
@@ -5,6 +5,5 @@
 description      'Implements an recipes for managing network resources on Lenovo switches'
 version          '1.0.0'
 long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-source_url       'https://github.com/arunktele/chef'
-issues_url       'https://github.com/arunktele/chef/issues'
-
+source_url       'https://github.com/lenovonetworking/chef-cnos'
```

Comment:
I suggest calling the cookbook directory something like `chef-cnos` not `chef`. People who use chef usually have a directory already called `chef` and this would confuse them.